config.go

```diff
@@ -54,30 +34,28 @@ type AdRequest struct {
 }
 
 type AdConfig interface {
-	NewAdRequest() *AdRequest
-	UpdateAdRequest(*AdRequest)
+	AdRequestData() *AdRequestData
+	UpdateAdRequest(*AdRequestData)
 	ServerUrl() string
 	AssetEndpointUrl() string
 	AssetEndpointDisplayAreas() []DisplayArea
 }
 
```

Comment:
Should I add setter methods to set this config property? 
like `SetUrl(url string)`  `SetAssetEndpointUrl(url string)`

Reply:
I will have to add the setter methods, will update the PR..